ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master.yaml

```diff
@@ -23,17 +23,24 @@ base_images:
     name: release
     namespace: openshift
     tag: golang-1.17
+  base:
+    name: "4.11"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.17-openshift-4.11:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.17-openshift-4.11
 build_root:
-  project_image:
-    dockerfile_path: Dockerfile.assisted-installer-build
+  from_repository: true
 images:
-- dockerfile_path: Dockerfile.assisted-installer
-  from: coreos-centos-stream8
+- dockerfile_path: Dockerfile.assisted-installer.ocp
+  from: base
   inputs:
-    openshift_release_golang-1.17:
+    ocp_builder_rhel-8-golang-1.17-openshift-4.11:
       as:
-      - registry.ci.openshift.org/openshift/release:golang-1.17
-  to: assisted-installer
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.11
+  to: agent-installer-orchestrator
 - dockerfile_path: Dockerfile.assisted-installer-controller
```

Comment:
Maybe this image is not needed here as well as we have the `edge` variant.

Reply:
Nit: `dockerfile_path: Dockerfile.assisted-installer-controller`

Reply:
The assisted-installer-controller needs to stick around. I know its confusing two containers come out of the assisted-installer repo.

The approved OCP payload name is `agent-installer-csr-approver` and file is `Dockerfile.assisted-installer-controller.ocp`